readme.md

```diff
@@ -20,8 +22,53 @@ Basically what I want is all the random online tools in one easy lightweight pla
     - Braille Translator
     - Semaphore Translator
     - Number/Hex to Letter
-- Improvements to wordhelper
+- Improvements to wordhelper (TODO)
     - Insert/Remove/Replace a string
-
-## Other Work
+- Anagram solver
+    - single word and multi-word anagram solver
+- Word search
+    - search a word with regex
 - We can put all these tools on a handy, simple website
+
+### Prerequisites
+
+(Once) Install the following items
+1. Python 3+
+1. [pipenv](https://docs.pipenv.org/)
+    ```
+    >pip install pipenv
+    ```
+
+### Setup
+
+(Once per code sync)
+Install all the dependencies by running
+```
+>.\setup.bat
+```
+
+### Usage
+
+(Once per use)
+Start `puzzle-utils` interactively by running
+```
+>.\run.bat
+>>> anagram("silent")
+['listen', 'inlets', 'tinsel', 'elints', 'silent', 'intels', 'enlist']
+```
+
+## Running the tests
+
+Explain how to run the automated tests for this system
+
+## Contributing
+
+Refer to [puzzle-utils/README.md](puzzle-utils/README.md) for details.
+
+## Authors
+
+See [contributors](https://github.com/s-zhang/puzzlehunt-tools/contributors).
+
+## License
+
+TBD
```

Comment:
May I suggest [GPL-3.0](https://opensource.org/licenses/GPL-3.0)

Reply:
This should be a separate discussion. See issue https://github.com/s-zhang/puzzlehunt-tools/issues/8